Automattic/sensei
# Set lesson status to ungraded after reset

Fixes #2245

#### Changes proposed in this Pull Request:

* Make sure lesson status is set to `ungraded`, after resetting and saving an already graded lesson. 


#### Testing instructions:

* Finish a quiz with some user that has auto-grade or grade it manually. 
* Open Review Grade, reset and save.
* Lesson status should turn back to Ungraded.
